src/fe-text/gui-entry.c

```diff
@@ -936,6 +936,26 @@ void gui_entry_set_pos(GUI_ENTRY_REC *entry, int pos)
 	gui_entry_draw(entry);
 }
 
+void gui_entry_set_text_and_pos_bytes(GUI_ENTRY_REC *entry, const char *str, int pos_bytes)
+{
+	int pos;
+	const char *ptr;
+
+        g_return_if_fail(entry != NULL);
```

Comment:
I see you're trying to stay consistent with the existing indentation style

Reply:
copy&paste of bad indent. fixed